### ISSUE TITLE
Sunset Overdrive (847370)

### ISSUE THREAD

**Issue by mimattr** [ValveSoftware/Proton#1989]
# Compatibility Report
- Name of the game with compatibility issues: **Sunset Overdrive**
- Steam AppID of the game: **847370**

## System Information
- GPU: **GTX 970**
- Driver/LLVM version: **NVIDIA 396.54**
- Kernel version: **4.4.0-139-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **3.16-4 Beta**

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

- [steam-847370.log](https://gist.github.com/mimattr/98181c6d7dc85b59d18088790922f22e)

## Symptoms 
Consistent application exits during launch process.

## Reproduction
1. Select the title within your `Library` using the Steam client and press the `Play` button

**Comment by nded03** [comment]
Game is really unstable on Proton. Usually it hangs before I can even reach the main menu. Disabling esync seems to improve it ever so slightly. It's a gamble whether I can make it in-game or not, and if I do there are some pretty obvious graphical and performance issues. Still, it does sort of work which is promising.

**Comment by aqxa1** [comment]
I've tested it with wine-staging-3.19 (with esync enabled) and DXVK git, and no crashes there, and none of the graphical glitches that are evident with Proton (black squares). I'm not sure if the performance is worse or not with Proton as I haven't been able to get in game recently at all there, but I suspect part of it is just shader and state caches being built.

EDIT: Bug also exists with Proton 3.7.8

**Comment by NTMan** [comment]
On Vega 56/64 the game has another issue:
- The game randomly stuck at splash screen.

![screenshot from 2018-11-29 00-17-46](https://user-images.githubusercontent.com/200750/49176836-92856280-f36d-11e8-9f21-ed4992886f2b.png)
![screenshot from 2018-11-29 00-19-35](https://user-images.githubusercontent.com/200750/49176855-987b4380-f36d-11e8-8547-78fd290c9da6.png)
![screenshot from 2018-11-29 00-22-33](https://user-images.githubusercontent.com/200750/49176857-9ca76100-f36d-11e8-8af8-7b4e2ba561c8.png)


## System Information
- GPU: AMD Vega 56/64
- Driver/LLVM version: Mesa 18.3.0-rc2/7.0.0
- Kernel version: 4.19.0-1
- Link to full system information report as [Gist](https://gist.github.com/NTMan/33cca4906328793206218192820779ec)
- Proton version: 3.16-4 beta

[steam-847370.log](https://github.com/ValveSoftware/Proton/files/2626309/steam-847370.log)

**Comment by mimattr** [comment]
# Whitelist Request
- Name of the game to be whitelisted: **Sunset Overdrive**
- Steam AppID of the game: **847370**

## System Information
- GPU: **GTX 970**
- Driver/LLVM version: **NVIDIA 415.25**
- Distro version: **Ubuntu 16.04.5 LTS**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **3.16-6 Beta**

## I confirm:
- [ ] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [x] that no workarounds other than PROTON_* env variables are necessary:
  - Add the `PROTON_NO_ESYNC=1 %command%` [launch option](https://github.com/ValveSoftware/Proton#runtime-config-options) within the Steam client.

## Issues
- [x] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues.

**Comment by kisak-valve** [comment]
@NTMan, does that launch option help the game on your system?

**Comment by NTMan** [comment]
> @NTMan, does that launch option help the game on your system?

Unfortunately it doesn't helps on my setup.
Log with enabled `PROTON_NO_ESYNC=1` option: [steam-847370.log](https://github.com/ValveSoftware/Proton/files/2799879/steam-847370.log)

**Comment by aqxa1** [comment]
Sunset Overdrive appears to work properly with Proton 4.11-1 (no need to disable esync).

**Comment by mimattr** [comment]
# Whitelist Request

## System Information
- GPU: **RTX 2060**
- Driver/LLVM version: **NVIDIA 430.40**
- Kernel version: **5.0.0-23-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **4.11-2**


## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [ ] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by kisak-valve** [comment]
Hello, is anyone seeing an issue with this game and Proton 4.11 that would block it from being whitelisted?

**Comment by aqxa1** [comment]
@kisak-valve Game still randomly GPU hangs with a Vega 56 and LLVM (as well as ACO). It's hard to debug because it looks to be random, and only happens every hour or so. Another GPU hang was fixed with ACO, but this one looks to be different as it occurs with both compilers.

**Comment by NTMan** [comment]
@kisak-valve I disagree for whitelisting this game because the game doesn't see XBOX Elite Controller, but works well with Dualshock 4 controller.

**Comment by hakzsam** [comment]
@aqxa1 Does the game still hang with recent LLVM/Mesa?

**Comment by aqxa1** [comment]
@hakzsam I don't have my Vega card currently installed, but I don't remember it hanging last time I played it with that GPU (but that is just from memory).

**Comment by McMarius11** [comment]
The game runs but after 15 minutes voices and music sometimes cut out. A restart of the game gets it back. This happens both during gameplay and cutscenes.
here is my log file 
[steam-847370.log](https://github.com/ValveSoftware/Proton/files/4437621/steam-847370.log)

if i have a new booted pc i can play 1 hour without any problems after that i get the same error.
someone on a forum posted this 
"I found a solution on the official forums. I guess I had to change my sound setting from HDMI 7.1 uncompressed to stereo uncompressed in the console's Display and Sound menu. If anyone has the same problem, that worked perfectly."
i don't know how can i do this

**Comment by Vash63** [comment]
DualSense controller doesn't work with Steam Input on this game, and also I'm getting an error that the save files were corrupted on first boot before any save files should have existed.

**Comment by LuanVSO** [comment]
game crashes some time after launch with esync and fsync enabled, disabling them fixes the crashes.

**Comment by SecularSteve** [comment]
I think the game can be re-evaluated by now.

Just marathoned halfway through it in the last 2 days with Proton on the Steam Deck (Maxed Out, 30 FPS), and never had an issue with it, fully OOTB aside from a few native bugs already present on Windows.

Just two pieces of advice:

1. Don't delete the corrupt savefile. It will crash the game (it's not corrupt, it's just some kind of placeholder).
2. Don't play around with the internal framerate settings. It may desync the audio.
